code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -73,6 +73,14 @@
 	var/wiresexposed = 0
 	var/locked = 1
 	var/has_power = 1
+
+	var/lawpreset = /datum/ai_laws/nanotrasen
+	var/spawn_module = null
+	var/key_type = null
+	var/spawn_sound = 'sound/voice/liveagain.ogg'
+	var/cell_type = /obj/item/weapon/cell/high
+	var/pitch_toggle = 1

```

Comment:
Whats the point of this var? I don't see anywhere its set to zero.

Personally i like the variable pitch, it adds a lot of variety even to mechanical sounds